CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 3.19...3.25)
 
+find_program(NVHPC_CXX_BIN "nvc++" REQUIRED)
+set(CMAKE_CXX_COMPILER ${NVHPC_CXX_BIN})
+
+find_program(NVHPC_C_BIN "nvc" REQUIRED)
+set(CMAKE_C_COMPILER ${NVHPC_C_BIN})
```

Comment:
This has to happen
Because cudecomp manually sets their compiler
Otherwise this causes an endless loop